xbmc/ServiceBroker.h

```diff
@@ -100,6 +99,11 @@ class CControllerManager;
 class CGameServices;
 } // namespace GAME
 
+namespace KEYBOARD
```

Comment:
Should this be grouped into an INPUT namespace?

Reply:
I considered this, but it would be very verbose - for classes outside KODI, a function parameter would be `KODI::INPUT::KEYBOARD::IWIndowKeymap`. I avoid using namespaces in other cpp files other than KODI and its own, so everywhere outside of input you'll have variables with type `INPUT::KEYBOARD::IWIndowKeymap` instead of the (already long) `KEYBOARD::IWIndowKeymap`.

From an architecture standpoint, INPUT would be beneficial, but I think it's too redundant for obvious input devices like keyboards for not much gain.